src/Raven.Server/Documents/Indexes/Static/IndexAndTransformerCompiler.cs

```diff
@@ -29,7 +29,7 @@ namespace Raven.Server.Documents.Indexes.Static
     [SuppressMessage("ReSharper", "ConditionIsAlwaysTrueOrFalse")]
     public static class IndexAndTransformerCompiler
     {
-        private static bool EnableDebugging = true; // for debugging purposes
+        private static bool EnableDebugging = false; // for debugging purposes (mind http://issues.hibernatingrhinos.com/issue/RavenDB-6960)
```

Comment:
Sorry, that was a bad copy / paste.
what I _meant_ to say is that it is better to add a test to ensure that this doesn't slit to prod